addons/starrocks-ce/templates/_helpers.tpl

```diff
@@ -74,3 +74,11 @@ be.conf: |
 {{ .Values.be.config | indent 2 }}
 {{- end }}
 {{- end }}
+
+{{- define "fe.componentDefName" -}}
+{{ include "starrocks.name" . }}-fe
```

Comment:
should we add a version for cmpd?

Reply:
StarRocks currently supports versions 3.2.2 and 3.3.0, both of which are compatible with the same cmpd API. We can consider adding version information to the cmpd name in the future when incompatible versions arise.

Reply:
Yeah, I mean add a version for the cmpd cr, not for the starrocks docker image version. So that in the future, if the cmpd CR upgrades, may be two versions of cmpd CR can co-exist.